modules/automod/automod.ts

```diff
@@ -14,6 +14,23 @@ import warn from "../punishments/warn.js";
 import tryCensor, { badWordRegexps, badWordsAllowed } from "./misc.js";
 import { stripMarkdown } from "../../util/markdown.js";
 import { joinWithAnd } from "../../util/text.js";
+import { createWorker } from "tesseract.js";
+
+const worker = await createWorker("eng");
+async function getMessageImageText(message: Message): Promise<string[]> {
+	const imageTextPromises = message.attachments
+		.filter((attachment) => attachment?.contentType?.match(/^image\/(bmp|jpeg|png|bpm|webp)$/i))
```

Comment:
remove ? after contentType to make eslint shut up

Reply:
Here's a tip: use \`\`\`suggestion
```suggestion
		.filter((attachment) => attachment.contentType?.match(/^image\/(bmp|jpeg|png|bpm|webp)$/i))
```